dlp/snippets/src/main/java/dlp/snippets/InspectDatastoreSendToScc.java

```diff
@@ -0,0 +1,124 @@
+  }
+
+  // Inspects a Datastore Table
+  public static void inspectDatastoreSendToScc(
+      String projectId, String datastoreNamespace, String datastoreKind)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DlpServiceClient dlpServiceClient = DlpServiceClient.create()) {
+
+      // Specify the Datastore entity to be inspected.
+      PartitionId partitionId =
+          PartitionId.newBuilder()
+              .setProjectId(projectId)
+              .setNamespaceId(datastoreNamespace)
+              .build();
+
+      KindExpression kindExpression = KindExpression.newBuilder().setName(datastoreKind).build();
+
+      DatastoreOptions datastoreOptions =
+          DatastoreOptions.newBuilder().setKind(kindExpression).setPartitionId(partitionId).build();
+
+      StorageConfig storageConfig =
+          StorageConfig.newBuilder().setDatastoreOptions(datastoreOptions).build();
+
+      // Specify the type of info the inspection will look for.
+      List<InfoType> infoTypes =
+          Stream.of("EMAIL_ADDRESS", "PERSON_NAME", "LOCATION", "PHONE_NUMBER")
+              .map(it -> InfoType.newBuilder().setName(it).build())
+              .collect(Collectors.toList());
+
+      // The minimum likelihood required before returning a match:
+      Likelihood minLikelihood = Likelihood.UNLIKELY;
+
+      // The maximum number of findings to report (0 = server maximum)
+      InspectConfig.FindingLimits findingLimits =
+          InspectConfig.FindingLimits.newBuilder().setMaxFindingsPerItem(100).build();
+
+      // Specify how the content should be inspected.
+      InspectConfig inspectConfig =
+          InspectConfig.newBuilder()
+              .addAllInfoTypes(infoTypes)
+              .setIncludeQuote(true)
+              .setMinLikelihood(minLikelihood)
+              .setLimits(findingLimits)
+              .build();
+
+      // Specify the action that is triggered when the job completes.
+      Action.PublishSummaryToCscc publishSummaryToCscc =
+          Action.PublishSummaryToCscc.getDefaultInstance();
+      Action action = Action.newBuilder().setPublishSummaryToCscc(publishSummaryToCscc).build();
+
+      // Configure the inspection job we want the service to perform.
+      InspectJobConfig inspectJobConfig =
+          InspectJobConfig.newBuilder()
+              .setInspectConfig(inspectConfig)
+              .setStorageConfig(storageConfig)
+              .addActions(action)
+              .build();
+
+      // Construct the job creation request to be sent by the client.
+      CreateDlpJobRequest createDlpJobRequest =
+          CreateDlpJobRequest.newBuilder()
+              .setParent(LocationName.of(projectId, "global").toString())
```

Comment:
nit: Consider moving the "global" to a variable declared in main()

Reply:
Generally, we are not declaring "global" in main() for all the code samples.
So if I make changes in current PR, we need to incorporate the changes for all the code sample thus I'm not making the requested changes.